### PR TITLE
Update docs to new import syntax and use the new release URL

### PR DESCRIPTION
**Pull request by ynotdraw**
#15723 here are the four items I picked from the issue.  I went through the jsdocs and replaced old usages with the new import syntax as well.  Let me know if this is overkill/should not be included here.  Those changes are the large majority of what changed.

### CODE REVIEW

File: packages/ember-runtime/lib/ext/function.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ if (ENV.EXTEND_PROTOTYPES.Function) {
     will instead clear the cache so that it is updated when the next `get`
     is called on the property.
 
-    See [Ember.ComputedProperty](/api/classes/Ember.ComputedProperty.html), [Ember.computed](/api/classes/Ember.computed.html).
+    See [ComputedProperty](/api/ember/release/classes/ComputedProperty), [Object.computed](/api/ember/release/classes/@ember%2Fobject%2Fcomputed).
```
Comment (toddjordan) [inline packages/ember-runtime/lib/ext/function.js]: I'd call it `[@ember/object/computed]` vs `[Object.computed]`

File: packages/ember-runtime/lib/utils.js [hunk 2]
```diff
@@ -86,26 +89,30 @@ export function isArray(obj) {
...
+  typeOf(101);                    // 'number'
+  typeOf(new Number(101));        // 'number'
+  typeOf(true);                   // 'boolean'
+  typeOf(new Boolean(true));      // 'boolean'
+  typeOf(A);                      // 'function'
+  typeOf([1, 2, 90]);             // 'array'
+  typeOf(/abc/);                  // 'regexp'
+  typeOf(new Date());             // 'date'
+  typeOf(event.target.files);     // 'filelist'
+  Ember.typeOf(EmberObject.extend());  // 'class'
```
Comment (toddjordan) [inline packages/ember-runtime/lib/utils.js]: typeOf for these 2 lines too?

**Comment by toddjordan** [comment]
This is awesome. Thanks!

File: packages/ember-runtime/lib/utils.js (hunk 2 above)
Comment (ynotdraw) [reply packages/ember-runtime/lib/utils.js]: Doh! Good catch!

**Comment by toddjordan** [comment]
Looks Great!  Thanks @ynotdraw !

**Comment by toddjordan** [comment]
@ynotdraw do you mind squashing these commits into one?  Its easier to cherry pick into the beta branch that way...

**Comment by ynotdraw** [comment]
Dangit - I messed up my squash as it pulled in other commits. Fixing again here in a few.

**Comment by mikemunsie** [comment]
@toddjordan Just for my learning as well, couldn't we  just use the `merge + squash` feature from github? Or do you guys prefer manually squashing the commits and then setting up a PR?

**Pull request closed without merging**